src/runtime_src/core/edge/user/system_linux.cpp

```diff
@@ -118,17 +144,19 @@ get_os_info(boost::property_tree::ptree &pt)
     }
     ifs.close();
   }
-
+  pt.put("model", machine_info());
+  pt.put("cores", std::thread::hardware_concurrency());
+  pt.put("memory", sysconf(_SC_PHYS_PAGES) * sysconf(_SC_PAGE_SIZE) / 0x100000);
```

Comment:
After reviewing the code base, I have come full circle to this entry.  Could you please do the following:

1. Change the "memory" entry name to "memory_bytes".
2. Remove the 0x100000 magic values.   It is best to keep the internal values in bytes, like we do with the other values.
3.  Add this value as a hex string value.  For example
`pt.put("memory_bytes", (boost::format("0x%x") %  sysconf(_SC_PHYS_PAGES) * sysconf(_SC_PAGE_SIZE).str().c_str())`

Reply:
Changes made to store memory as as hex string.